.github/workflows/deploy.yml

```diff
@@ -0,0 +1,46 @@
+name: Surge Deploy
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Deploying to surge
+    steps:
+      - uses: actions/checkout@v2
+      - name: restore lerna
+        uses: actions/cache@v2
+        with:
+          path: |
+            node_modules
+            */*/node_modules
+          key: ${{ runner.os }}-${{ hashFiles('**/yarn.lock') }}
+
+      - name: Fire deployment
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
+
+      - name: Trying echo
+        run: echo "${GITHUB_REF#refs/heads/}"
+
+      - name: Install lerna
+        run: npm install -g lerna
```

Comment:
If this is for `lerna bootstrap`, Lerna is already a dependency of this project.